config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -310,11 +310,63 @@ periodics:
           value: "standard"
         - name: CONTROL_PLANE_MACHINE_COUNT
           value: "3"
+        - name: KUBERNETES_VERSION
+          value: 1.23.5
```

Comment:
We should be using the latest supported version of Kubernetes supported by capz to test CCM/CNM at cloud-provider-azure:master, updated this here while I was working on this new periodic job